src/.vuepress/theme/styles/layout/Menu Docs.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you please remove this file?  
Maybe you can add a new rule to the `.gitignore` file at the repository root to avoid committing this kind of file related to your local configuration